.travis.yml

```diff
@@ -8,7 +8,7 @@ deploy:
   provider: npm
   email: barlockm@gmail.com
   api_key:
-    secure: UrRKJhYIb86Ygq5i/yTd/OiU/vObGLtisclhANCqRGpARTWQ6lWiYUehP2YwGdUntcEnByeLS8dS8LlMYY4OOZL6aq9YDRJLEFtoeNumIVFtb+F+6yFex9mwgEcTY0gfwlipp4VYIHBUaoMKROHRfKc2C/4UKHLq+9+Al98yavPjK4SEg/lHdAjWtRjTZUD35daqNwiegyzP6JF25D/uMUiWW9J1ZcaiqRb4uKqfdt66G2lc81zw+waQNVl9le63kq5xV25K6MFT1JY7UQgi/8jp15flXdUNvZLN85Z00lP29a+PeXFjk6Oms+fs7NhHf7WMFx6Hcwtv07N5dXsjZlhOZKZ1Lf6Q+XkP1shydLO5e4MRHSLmqZI17/iCXVL7K99QdjN+qFqHEnr0rgm7PMQt8cHlaanrPjNyqm9HadNXPkUBWcsKpXR4EuHJH9kgb5bHVF+pWkbXVMRe1c70XKH1AI0K3MCeCo94S2OIYAhmpP2YJCqOQepNxzgXrPV/2fkL0MfgBE0ltVPbrqQv3SAW963TK0yNi+KPyOzObYJOBU0SwCkqNwXOuO3/W9irFbQSrP60Ru8q33goAPWq6t+Rt9TonEzahlNYQ0M+6oKtjQUNQjeW5IaP7ozNg8ECsaHYAM1CURhuEPdwlkwL+KVIzm6SvhR2zyaLS1k+sgk=
+    secure: PbQt7nedejcNpePz67XMSmGxFKNFIPDrvcVKtrhgUKJFTsL8Yg3FY+Xosn5Zoz4uqG7ayC664IR1cywg+x+gjDdwu/LW3FwllqVpsdtbp360agzYrRTOs257lH6VFCy+EAOemXxmaDEOST115E9ron6UtK54P9BKyDR9Qg9tUHVPzKcL6OT3DdDan3uDUY+TLAo+M7hT9dqb2qx2oh4XVvy9O+NqytsmEgSyo4PboieeJk01MM/j9EaCZWU2fJnv7eX3CZfR2j+9fwemP9kPeCuK4UkeGe1PcFLj3HMtiHwHw+rCcnih9R/ZYAZXck9OVxqsnadorGSYK9FLrGwEIloLaAcak3wo8xBizvFEYAlPjJyPdzSVo+2tO42znUXpq4cT97+ba/tEQbTbaQj/55TsQH0v3tTJ7lZX8/lJRmCIzauRv8z2/DdZYQ/pi2rq82vkjV3ovp9oI2dmA7U4hun+WHXApSSFIgALf3fooTxrFZr6r9y0s/Nhjp7SNssmhCz+kO3pt+El+rI0OOsfTHm/Nef+2e2GMUmKMOzYeAtKZGeycQVecunpsmxdhmj91An2CIUvZtICsAtJ3Vm1K938A6YhCW5um8GDjzn479idydX1KYi41Cxy7RQheDd0mPxm1elz9zq/OH9nCTD4fDgh7SUEHsACMsG1EpdF8rI=
```

Comment:
Releases haven't been working. I realized this is because I was using the wrong travis endpoint. Giving this a shot and hoping.